opencog/cython/opencog/atomspace_details.pyx

```diff
@@ -411,8 +435,10 @@ cdef api object py_atomspace(cAtomSpace *c_atomspace) with gil:
     cdef AtomSpace atomspace = AtomSpace_factory(c_atomspace)
     return atomspace
 
-cdef api object py_atom(UUID uuid, object atomspace):
-    cdef Handle temphandle = Handle(uuid)
-    cdef Atom atom = Atom(temphandle, atomspace)
+cdef api object py_atom(void* atom_ptr, object atomspace):
+    cdef atom_ptr_long = PyLong_FromVoidPtr(atom_ptr)
+    cdef Handle temp_handle = Handle(None, atom_ptr_long)
```

Comment:
This line is the problematic line. You want a Python Handle object to be created, and you must initialize it with Python object arguments only. In this case, I'm initializing it with the None singleton Python object, and a python number object that contains the address of the shared pointer which will, in turn, be passed into the C++ cHandle object's constructor.

Perhaps I should go back to my original variable name, `atom_ptr_ptr`? I thought that was equally confusing because then one wonders what it means when the code uses `atom` to refer to objects of type AtomPtr, then atom_ptr_ptr seems like there is one more layer of indirection than there really is.